CRM/Cplabels/Form/Search/Cplabels/CommonGround.php

```diff
@@ -18,18 +19,15 @@ public function buildForm(&$form) {
     $this->setTitle(ts('Mailing Labels: Common Ground'));
 
     // This search has no editable criteria.
-    $form->addElement('static', 'statichtml', '', '<p class="status alert">'. E::ts('This search has no editable criteria. Please click "Search".') . '</p>');
+    $form->addElement('static', 'statichtml', '', '<p class="status alert">' . E::ts('This search has no editable criteria. Please click "Search".') . '</p>');
     $form->assign('elements', array('statichtml'));
   }
 
   /**
    * Get a list of summary data points
    *
-   * @return mixed; NULL or array with keys:
```

Comment:
I'd like to keep the `@return` documentation here. Please see my comments above about the similar topic. Thanks!